mantis-common/build.gradle

```diff
@@ -41,6 +40,8 @@ dependencies {
     api libraries.slf4jLog4j12
     implementation libraries.commonsIo
     implementation 'net.jcip:jcip-annotations:1.0'
+    implementation libraries.micrometerCore
+    implementation libraries.spectatorApi
```

Comment:
Is this needed because we are dual publishing the metrics?

Reply:
Yes, the spectator library can be removed when all the metrics have been changed.